src/ViewInstance.js

```diff
@@ -173,6 +175,13 @@ class ViewInstance extends React.Component {
     });
   };
 
+  selectInstanse = (instance) => {
+    const { location: { search } } = this.props;
+
+    this.setState({ findInstancePluginOpened: false });
```

Comment:
Does it make sense to set the state if the next line is going to navigate away from the page? If it is I would call the navigation in the callback which is the second argument to the `setState` as eventually, it is possible that the warning pop up saying that state is going to be set on the unmounted component.

Reply:
It looks like setState is unnecessary here